docs/concepts/vega-protocol.md

```diff
@@ -2,69 +2,273 @@
+
+Further reading: [EIP-20: Token Standard proposal](https://eips.ethereum.org/EIPS/eip-20)
 :::
 
+After a successful deposit transaction, the `Asset_Deposited` event will be emitted for use by the Vega event queue.
+
+The transaction is recognised by the Vega event queue and packaged as an event, which is then submitted to the validator nodes to verify the event contents against an Ethereum node that Vega validators also run. 
+
+Once the transaction is verified, the Vega public key submitted in the transaction will be credited with the deposited asset.
+
+**Read more**: [ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#deposit_asset)
+
+### Diagram: Deposits
+![Deposit diagram](/img/concept-diagrams/diagram-deposit.png)
+
 ### Withdrawals
-In restricted mainnet, the only assets that are available to withdraw are rewards accrued through staking tokens to network validators.
+Assets used for trading and related activities can only be withdrawn if they are not being held in bond for liquidity or in the margin account for active orders. VEGA tokens can only be withdrawn if they are not staked.
+
+When a participant decides they want to remove their assets from the Vega network, they'll need to submit a withdrawal request via a Vega app or the API.
+
+This request, if valid, will put through Vega consensus, wherein the validators will sign a multi-signature withdrawal order bundle, and assign an expiry. If the withdrawal is not completed by the participant before the expiry, the tokens are returned to their collateral account. 
 
-To stake those rewards, the recipient must withdraw them from the Vega key the rewards are credited to, and send the tokens to their Ethereum wallet. After withdrawing, the tokens can be associated with a Vega key.
+To move the assets into the participant's Ethereum wallet, they need to submit the bundle to the ERC20 bridge. The bridge validates the bundle and then releases the funds to the Ethereum wallet.
 
-Note: Associated and deposited are not equivalent, as deposited tokens are held within the ERC20 bridge contract, and associated tokens stay in an Ethereum wallet or in the vesting contract.
+Once a successful withdrawal transaction has occurred, the ERC20 bridge will emit an `Asset_Withdrawn` event, and confirms to the Vega network that the withdrawal has been completed.
+
+**Read more**: [ERC20 bridge logic API documentation](../api/bridge/contracts/ERC20_Bridge_Logic.md#withdraw_asset)
+
+### Diagram: Withdrawals
+![Withdrawal diagram](/img/concept-diagrams/diagram-withdraw.png)
+
+##### Withdrawing VEGA
+VEGA (an ERC20 token) used for staking is associated with a Vega key, and so to withdraw those tokens, they must be dissociated first.
```

Comment:
Do you mean withdraw as in withdraw from bridge here? If not is the double meaning of withdraw confusing?